index.html

```diff
@@ -4,10 +4,24 @@
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <title>Document</title>
+   <title>Toshia_moi_mentor_dorogoi</title>
```

Comment:
pick the proper name for your pet project,

Reply:
done

Reply:
it is still the old name